src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+
+public class Deadline extends Task {
```

Comment:
It would be better to include a comment on what the class is about at the top of the class